applications/DamApplication/python_scripts/dam_construction_utility.py

```diff
@@ -66,14 +66,12 @@ def Initialize(self):
     def InitializeSolutionStep(self):
 
         time = self.mechanical_model_part.ProcessInfo[TIME]
-        delta_time = self.mechanical_model_part.ProcessInfo[DELTA_TIME]
-        step = int(time/delta_time)-1
-
+        tol = 1e-10
```

Comment:
Well, instead of defining a tolerance of 1.0e-10, i would define something like:
tol = self.mechanical_model_part.ProcessInfo[DELTA_TIME]*1.0e-10